pkg/shell/indexes.js

```diff
@@ -458,6 +458,8 @@ var phantom_checkpoint = phantom_checkpoint || function () { };
                         .attr("class", "fa-li fa fa-circle")
                         .css("color", machine.color || ""));
                 return $("<li role='presentation'>")
+                    // FIXME: This isn't updated on switch
+                    .toggleClass("active", machine.key == machines.lookup()['key'])
```

Comment:
I'm pretty sure one or both of the following is true:

1. This isn't the proper way to check the current machine versus the one in the list
2. This probably is called to create the list once, not every time (didn't check)

I left this in as a FIXME as it adds the 'active' class and shows the current machine on page load at least. I figure you could do a more proper fix and we can squash it together later.